packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -197,6 +197,15 @@ export default abstract class EasCommand extends Command {
       cleanMessage += err?.graphQLErrors?.[0]?.extensions?.requestId
         ? `\nRequest ID: ${err.graphQLErrors[0].extensions.requestId}`
         : '';
+
+      if("UNAUTHORIZED_ERROR" === err?.graphQLErrors?.[0]?.extensions?.errorCode){
+        cleanMessage = "Your Expo account may not have the minimum permission to do this operation." +
+          'For more information visit: https://github.com/expo/expo/issues/22696';
+
+        Log.error(cleanMessage);
+        throw new Error(cleanMessage);
```

Comment:
This is the `catch` function, so we shouldn't throw a new error here, but rather handle the error that this function received.